mongodb-atlas-mixin/config.libsonnet

```diff
@@ -1,18 +1,45 @@
 {
-  _config+:: {
-    // sharding dashboard flag
-    enableShardingOverview: false,
+  local this = self,
 
-    dashboardTags: ['mongodb-atlas-mixin'],
-    dashboardPeriod: 'now-30m',
-    dashboardTimezone: 'default',
-    dashboardRefresh: '1m',
+  // Basic filtering - MongoDB Atlas uses job and cl_name (cluster name) as primary filters
+  filteringSelector: 'job="integrations/mongodb-atlas"',
+  groupLabels: ['job', 'cl_name'],
+  instanceLabels: ['instance'],
 
-    // alerts thresholds
-    alertsDeadlocks: 10,  // count
-    alertsSlowNetworkRequests: 10,  // count
-    alertsHighDiskUsage: 90,  // percentage: 0-100
-    alertsSlowHardwareIO: 3,  // seconds
-    alertsHighTimeoutElections: 10,  // count
+  // Dashboard settings
+  dashboardTags: ['mongodb-atlas-mixin'],
+  uid: 'mongodb-atlas',
+  dashboardNamePrefix: 'MongoDB Atlas',
+  dashboardRefresh: '1m',
+  dashboardPeriod: 'now-30m',
+  dashboardTimezone: 'default',
+
+  // Sharding dashboard flag, enable this to generate the sharding overview dashboard
+  enableShardingOverview: true,
+
+  // Logs configuration (MongoDB Atlas does not have Loki logs by default)
+  enableLokiLogs: false,  // note for users, this is not supported by the MongoDB Atlas mixin as there shouldn't be any logs to monitor yet
+  logLabels: [],
+  extraLogLabels: [],
+  logsVolumeGroupBy: 'level',
+  showLogsVolume: false,
+
+  // Alert thresholds with units
+  alertsDeadlocks: 10,  // count
+  alertsSlowNetworkRequests: 10,  // count
+  alertsHighDiskUsage: 90,  // %
+  alertsSlowHardwareIO: 3,  // seconds
```

Comment:
Like I commented on a previous PR, we could consider having the units be more tightly coupled with the native metric unit, e.g. milliseconds in order to simplify the query